components/tidb_query/src/codec/mysql/time/extension.rs

```diff
@@ -147,6 +148,31 @@ impl DateTimeExtension for Time {
     }
 }
 
+pub trait MonthExtension {
```

Comment:
Thanks! Could you change the non-vectorized version as well?

Reply:
I have modified